detection_rules/devtools.py

```diff
@@ -83,7 +83,11 @@ def update_lock_versions(rule_ids):
     if not click.confirm('Are you sure you want to update hashes without a version bump?'):
         return
 
-    rules = RuleCollection.default().filter(lambda r: r.id in rule_ids)
+    rules = RuleCollection.default()
+
+    if "*" not in rule_ids:
```

Comment:
Might be more intuitive usage to just remove `required=True` from the argument and change this to

```suggestion
    if rule_ids:
```

Reply:
yeah good point. we have the `confirm` anyway, so that sounds good.